src/libsync/CMakeLists.txt

```diff
@@ -96,7 +100,7 @@ target_compile_definitions(libsync
             QT_NO_FOREACH
 )
 
-target_link_libraries(libsync PUBLIC OpenAPI::LibreGraphAPI)
+target_link_libraries(libsync PUBLIC $<BUILD_INTERFACE:OpenAPI::LibreGraphAPI>)
```

Comment:
See above, you probably should add an install interface include.

Reply:
Its not ready for use yet, and the 3 headers  we install don't depend on it.

Reply:
Might be a bit hacky, its basically a PRIVATE lib and I was to lazy to explicitly link the ui to it..